snode-src/duckcoin.go

```diff
@@ -220,12 +223,13 @@ func loadDifficultyAndURL() {
 	_ = r.Body.Close()
 }
 
+// duckToAddress converts a Duckcoin public key to a Duckcoin address.
 func duckToAddress(duckkey string) string {
 	hash := sha256.Sum256([]byte(duckkey))
 	return b64(hash[:])
 }
 
-// create a new block using previous block's hash
+// makeBlock creates a new block using previous block's hash
```

Comment:
```suggestion
// makeBlock creates one new block by accepting the last block on blockChan, and restarting mining in case a new block is sent. It takes in the user's private key to be used in signing tx, the transaction, if tx.Amount is not 0. It also takes in the arbitrary data to be included in the block and the user's address (solver).
```